ReactiveXComponent/RabbitMq/RabbitMqSubscriber.cs

```diff
@@ -19,112 +19,128 @@ public class RabbitMqSubscriber : IXCSubscriber
         private readonly IXCConfiguration _xcConfiguration;
         private readonly string _privateCommunicationIdentifier;
 
-        private readonly Dictionary<SubscriberKey, RabbitMqSubscriberInfos> _subscribers;
-        private readonly Dictionary<SubscriberKey, List<Action<MessageEventArgs>>> _listenerToUpdateBySubscriberKeyRepo;
+        private readonly ConcurrentDictionary<SubscriberKey, RabbitMqSubscriberInfos> _subscribers;
+        private readonly ConcurrentDictionary<SubscriberKey, ThreadSafeList<Action<MessageEventArgs>>> _listenerToUpdateBySubscriberKeyRepo;
```

Comment:
If I manage thread safety at the dico level I wouldn't be able to work on the different dico's values (lists) at the same time.